schedule/migration/migration_offline.yaml

```diff
@@ -0,0 +1,36 @@
+name:           x86_test_offline.yaml
```

Comment:
I think it could be migration_offline_x86_64.yaml, please update the file name also.

Reply:
I updated it, please help to review, thanks.